### PR TITLE
Fixed external links in devices pages

### PR DESCRIPTION
**Pull request by aswinshenoy**
I replaced <LINK with <a for external links in the [chat.susi.ai/devices](https://chat.susi.ai/devices) page.

Fixes the issue #1662 I raised completely, and the links work as expected instead of returning 404.

Changes: [Add here what changes were made in this issue and if possible provide links.]

Demo Link: https://pr-1670-fossasia-susi-web-chat.surge.sh/devices

Screenshots for the change: 
BEFORE
image
AFTER
image

### CODE REVIEW

**Comment by rajvaibhavdubey** [comment]
@aswinshenoy the links are not correct, please correct them.
screenshot from 2018-11-01 16-44-19

**Comment by aswinshenoy** [comment]
@rajvaibhavdubey I fixed the links :) the links had got outdated, google play changed their format.

File: src/components/Devices/Devices.react.js [hunk 1]
```diff
@@ -54,14 +54,10 @@ class Devices extends Component {
               to have access to SUSI on the go.
             </p>
             <div className="store_icons">
-              <Link
-                className="play_store"
-                target="_blank"
-                to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"
-              >
+              <a className="play_store" href="https://play.google.com/store/apps/details?id=ai.susi">
```
Comment (akshatnitd) [inline src/components/Devices/Devices.react.js]: Please add `target="_blank"`

File: src/components/Devices/Devices.react.js [hunk 2]
```diff
@@ -77,14 +73,10 @@ class Devices extends Component {
               access to SUSI on the go.
             </p>
             <div className="store_icons">
-              <Link
-                className="app_store"
-                target="_blank"
-                to="https://github.com/fossasia/susi_iOS"
-              >
+              <a className="app_store" href="https://github.com/fossasia/susi_iOS">
```
Comment (akshatnitd) [inline src/components/Devices/Devices.react.js]: Please add `target="_blank"`

File: src/components/Devices/Devices.react.js (hunk 1 above)
Comment (aswinshenoy) [reply src/components/Devices/Devices.react.js]: I have added target="_blank"

File: src/components/Devices/Devices.react.js (hunk 2 above)
Comment (aswinshenoy) [reply src/components/Devices/Devices.react.js]: @akshatnitd I haved added target="_blank".

**Comment by aswinshenoy** [comment]
@akshatnitd I have resolved the conflicts :)

**Comment by AakashMallik** [comment]
@aswinshenoy Run your code at least once before pushing. You forgot to close a div tag.

**Comment by AakashMallik** [comment]
@aswinshenoy Pass the code through a linter before pushing.

**Comment by garvitkhandelwal** [comment]
@AakashMallik  Can I send a pull request for this issue or it is already solved?

**Pull request closed without merging**